doc/classes/EditorSettings.xml

```diff
@@ -381,6 +381,8 @@
 		<member name="editors/3d/selection_box_color" type="Color" setter="" getter="">
 			The color to use for the selection box that surrounds selected nodes in the 3D editor viewport. The color's alpha channel influences the selection box's opacity.
 		</member>
++		<member name="editors/3d_gizmos/gizmo_colors/aabb" type="Color" setter="" getter="">
```

Comment:
```suggestion
		<member name="editors/3d_gizmos/gizmo_colors/aabb" type="Color" setter="" getter="">
```
You included the `+` character, remove both, also please add a description

Reply:
Oops! I was commenting on the same thing at the same time!